crypto/store/loader_file.c

```diff
@@ -10,6 +10,7 @@
 #include <string.h>
 #include <sys/stat.h>
 #include <assert.h>
+#include <ctype.h>
```

Comment:
It was a missing header for Windows builds, that presumably was being loaded indirectly.
I'll change it to ossl_tolower.